README.md

```diff
@@ -57,6 +57,7 @@ A small visualization of the weekly markets in different cities. Available at: h
 |[München][muenchen-wikipedia]|[City of München][muenchen-markets]|
 |[Münster][muenster-wikipedia]|[City of Münster][muenster-markets]|
 |[Neu-Ulm][neu-ulm-wikipedia]|[City of Neu-Ulm][neu-ulm-markets]|
+|[Nersingen][nersingen-wikipedia]|[City of Nersingen][nersingen-markets]|
```

Comment:
Please keep the alphabetic order of the list. Same at the links section at the bottom of the file. Feel free to overwrite this commit via `git commit --amend` and `git push --force origin market-data-nersingen`.